src/libstd/fs.rs

```diff
@@ -348,6 +348,18 @@ impl File {
             inner: self.inner.duplicate()?
         })
     }
+
+    /// Reads a number of bytes starting from a given offset.
```

Comment:
It's specifically an offset from the start of the file, not the current cursor position, right? Could you add that to the docs?